about.qmd

```diff
@@ -69,7 +69,7 @@ Code in this repo is provided without warranty or promise of support. This is no
 ## Viewing CosMx data with Napari
 
 - [X] [Intro: using Napari to view and analyze CosMx data and creating napari-ready files from AtoMx exports](posts/using-napari-for-cosmx-data/index.qmd)
-- [ ] Napari-CosMx plugin basics
+- [X] [Napari-CosMx plugin essentials](posts/napari-cosmx-basics/using-napari-cosmx.qmd)
```

Comment:
It should work without index.qmd but I will rename it (at the end of the edits) to in case it still has issues. In other settings (not scratch space), I am pretty sure I've linked files that were not index.qmd without issue.